R/classes.R

```diff
@@ -1,11 +1,13 @@
-# Constructor functions for objects of fundamental rbabylon classes
-
+#' Create bbi object
+#'
+#' Constructor functions for objects of fundamental rbabylon classes.
+#' @name create_bbi_object
+NULL
 
-#' Creates S3 object of class `bbi_{.model_type}_model` from list with `MODEL_REQ_INPUT_KEYS`
+#' @describeIn create_bbi_object Creates list object of class `bbi_{.model_type}_model` from named list with `MODEL_REQ_INPUT_KEYS`
```

Comment:
Everything to be documented in `create_bbi_object` also has `@keywords internal`, which means that `?create_model_object` (for example) shows the documentation, but this documentation page is not included in the index. Just confirming that is the desired behavior.

Reply:
yes, these are all private methods so no need to include them in the index. They're for developers only.